olympus-frontend/vi/messages.po

```diff
@@ -1784,152 +1784,152 @@ msgstr "hrs"
 msgid "sOHM"
-msgstr ""
+msgstr "sOHM"
 
 #: src/views/Give/RecipientModal.tsx:450
 msgid "sOHM Allocation"
-msgstr ""
+msgstr "Phân bổ sOHM"
 
 #: src/views/Stake/Stake.tsx:457
 #: src/views/Stake/Stake.tsx:518
 #: src/views/V1-Stake/V1-Stake.jsx:386
 #: src/views/V1-Stake/V1-Stake.jsx:416
 #: src/views/Wrap/Wrap.tsx:281
 msgid "sOHM Balance"
-msgstr ""
+msgstr "Số dư sOHM"
 
 #: src/views/Stake/Stake.tsx:542
 msgid "sOHM Balance in Fuse (v1)"
-msgstr ""
+msgstr "Số dư sOHM trong Fuse (v1)"
 
 #: src/components/GiveProject/ProjectCard.tsx:262
 #~ msgid "sOHM Donated"
-#~ msgstr ""
+#~ msgstr "sOHM đã quyên góp"
 
 #: src/components/GiveProject/ProjectCard.tsx:274
 #~ msgid "sOHM Goal"
-#~ msgstr ""
+#~ msgstr "Mục tiêu sOHM"
 
 #: src/views/33Together/PoolInfo.tsx:112
 msgid "sOHM Pool Details"
-msgstr "Chi tiết Nhóm Pool"
+msgstr "Chi tiết bể sOHM"
 
 #: src/views/33Together/PoolInfo.tsx:108
 msgid "sOHM Prize Pool"
-msgstr "Giải thướng sOHM"
+msgstr "Tổng giải thướng sOHM"
 
 #: src/components/GiveProject/ProjectCard.tsx:290
 msgid "sOHM Yield"
-msgstr ""
+msgstr "Lợi suất sOHM"
 
 #: src/components/GiveProject/ProjectCard.tsx:302
 msgid "sOHM Yield Goal"
-msgstr ""
+msgstr "Lợi suất sOHM mục tiêu"
 
 #: src/views/Give/RecipientModal.tsx:533
 msgid "sOHM deposit"
-msgstr ""
+msgstr "gửi sOHM"
 
 #: src/components/GiveProject/ProjectCard.tsx:236
 #~ msgid "sOHM raised"
-#~ msgstr ""
+#~ msgstr "sOHM đã huy động"
 
 #: src/views/33Together/PoolPrize.tsx:169
 msgid "sec"
-msgstr "số giây"
+msgstr "giây"
 
 #: src/components/RebaseTimer/RebaseTimer.tsx:71
 msgid "to next rebase"
-msgstr "Cho đến khi thay đổi tiếp theo"
+msgstr "Cho đến kỳ tái cơ sở tiếp theo"
 
 #: src/views/BondV2/ChooseBondV2.tsx:99
 #~ msgid "v1 Bonds"
-#~ msgstr ""
+#~ msgstr "Trái phiếu v1"
 
 #: src/views/ChooseBond/ChooseBond.tsx:87
 msgid "v2 bonds"
-msgstr ""
+msgstr "Trái phiếu v2"
 
 #: src/views/Give/WithdrawDepositModal.tsx:95
 msgid "withdrawn"
-msgstr ""
+msgstr "rút"
 
 #: src/views/Stake/Stake.tsx:526
 #: src/views/V1-Stake/V1-Stake.jsx:401
 #: src/views/Wrap/WrapCrossChain.tsx:209
 msgid "wsOHM Balance"
-msgstr ""
+msgstr "Số dư wsOHM"
 
 #: src/views/V1-Stake/V1-Stake.jsx:409
 msgid "wsOHM Balance in FiatDAO"
-msgstr ""
+msgstr "Số dư wsOHM tại FiatDAO"
 
 #: src/views/Stake/Stake.tsx:534
 msgid "wsOHM Balance in FiatDAO (v1)"
-msgstr ""
+msgstr "Số dư wsOHM tại FiatDAO (v1)"
 
 #: src/views/TreasuryDashboard/components/Metric/Metric.js:84
 #~ msgid "wsOHM Price"
-#~ msgstr ""
+#~ msgstr "Giá wsOHM"
 
 #: src/views/Give/RecipientModal.tsx:314
 #~ msgid "{0} by {1}"
-#~ msgstr ""
+#~ msgstr "{0} bơi {1}"
 
 #: src/views/Give/RecipientModal.tsx:569
 #~ msgid "{0} sOHM"
-#~ msgstr ""
+#~ msgstr "sOHM {0}"
 
 #: src/views/Give/RecipientModal.tsx:522
 #~ msgid "{depositAmount} sOHM"
-#~ msgstr ""
+#~ msgstr "sOHM {depositAmount}"
 
 #: src/components/GiveProject/ProjectCard.tsx:366
 #~ msgid "{details}"
-#~ msgstr ""
+#~ msgstr "{chi tiết}"
```

Comment:
the build error is caused by syntax on this line

Reply:
@biwano separately from this PR... I'm wondering why this even shows up as a translatable string? `src/components/GiveProject/ProjectCard.tsx:366` on develop does not include this string.

The string `{details}` appears on line 511, but it does not have a `t` translations call out either.

Reply:
1. src/components/GiveProject/ProjectCard.tsx:366
msgstr "{chi tiết}" into msgstr "{details}"
2. src/components/EducationCard.tsx:235
msgstr "{tin nhắn}" into msgstr "{message}"

Reply:
Hi @appleseed-iii 
I copy some of our conversation here for future reference.

```The problematic lines are 
#: src/components/GiveProject/ProjectCard.tsx:366
#~ msgid "{details}"
#~ msgstr ""
#~ msgstr "{chi tiết}"```

He translated the variables. And since there are special caracters it completly breaks lingui commands.

I think at one point a developper wrote it (the t or Trans item). `lingui extract` produced the lines then the developper removed it and `lingui extract commented` the lines out.

Reply:
Hi @appleseed-iii Can I close this pull request? I realized we should have solved the issue in this PR and merged it back but I missed the point and reintegrated the changes manually.